eth/sync.go

```diff
@@ -216,15 +216,20 @@ func (cs *chainSyncer) modeAndLocalHead() (downloader.SyncMode, *big.Int) {
 			return downloader.SnapSync, td
```

Comment:
We still can go to snap sync in this case, right? I think this was the reason due to which we commented it earlier. Can we just return `downloader.FullSync, td` right away in the beginning?